build.sbt

```diff
@@ -3,37 +3,30 @@ import Testing._
 
 val compileAndTest = "compile->compile;test->test"
 
-lazy val workbenchUtil = project.in(file("util"))
-  .settings(utilSettings:_*)
-  .withTestSettings
-
 lazy val rawlsModel = project.in(file("model"))
   .settings(modelSettings:_*)
   .withTestSettings
 
 lazy val workbenchMetrics = project.in(file("metrics"))
   .settings(metricsSettings:_*)
-  .dependsOn(workbenchUtil % compileAndTest)
+  .dependsOn(rawlsModel)
```

Comment:
Possibly. I found a dependency on subsystem in StatsDTestUtils, but that might be an unused import.